primitives/core/src/offchain/mod.rs

```diff
@@ -28,6 +28,9 @@ pub mod storage;
 #[cfg(feature = "std")]
 pub mod testing;
 
+/// Local storage prefix used by the Offchain Worker API to
+pub const STORAGE_PREFIX : &'static [u8] = b"storage";
```

Comment:
@tomusdrw ain't this too generic? although I think it is too late to change this.

Reply:
It is stored in a separate rocksdb column anyway. This is just an extra prefix to distinguish with fork-aware storage and potentially other things in the future.